### ISSUE TITLE
Aria 1.1 Combo box not getting focused with Single Button navigation shortcuts 

### ISSUE THREAD

**Issue by saadataz** [nvaccess/nvda#11047]
NVDA does not jump to edit fields defined using [Aria 1.1 Combobox syntax](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html). However, it works fined with [Aria 1.0 Combo box examples](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.0pattern/combobox-autocomplete-both.html)

### Steps to reproduce:
1. Open chrome
1. Navigate to https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html
1. Press tab to bring focus to the page
1. Press E

### Actual behavior:
Pressing shortcut E or C does not focus the input box
NVDA announces "No next edit field"

### Expected behavior:
Pressing keyboard shortcut E should focus the input box

#### NVDA installed/portable/running from source:
portable
#### NVDA version:
2019.2.1
#### Windows version:
Windows 10 Pro
10.0.18363
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no 
#### If addons are disabled, is your problem still occuring?
yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
no

**Comment by Adriani90** [comment]
@saadataz I cannot reproduce this with NVDA 2019.3.1 on Google Chrome 81.
Please test NVDA 2019.3.1 on newest Chrome version.
And also please run the COM registration fixing tool in NVDA menu under "tools" before reproducing this issue.

**Comment by saadataz** [comment]
Just tested with NVDA 2019.3.1 and Chrome 80. Its working now
Thanks!
